src/components/jumbotron/jumbotron.js

```diff
@@ -97,14 +97,14 @@ export const BJumbotron = /*#__PURE__*/ Vue.extend({
         h(
           props.leadTag,
           { staticClass: 'lead' },
-          normalizeSlot('lead', {}, $scopedSlots, $slots) || props.leadHtml || stripTags(props.lead)
+          normalizeSlot('lead', $scopedSlots, $slots) || props.leadHtml || stripTags(props.lead)
```

Comment:
`{}` is the scope for normalizeSlot... it can't be skipped like `h(tag, {}, ...)` can.
So what is now happening is that it is seeing `$scopedSlots` as the scope to pass to the function, and seeing `$slots` as the `$scopedSlots` function.

Reply:
Good catch, this is a mistake.